services/course-material/src/components/Page.tsx

```diff
@@ -38,7 +38,7 @@ const Page: React.FC<Props> = ({ onRefresh, organizationSlug }) => {
             organizationSlug={organizationSlug}
           />
         )}
-      <SelectCourseInstanceModal onClose={onRefresh} />
```

Comment:
Fix the "No course id defined" error message that pops up if exam is not tied to a course